NativeEmptyExec should fallback on unsupported types

```
Caused by: scala.NotImplementedError: Data type conversion not implemented DateTimeType
        at org.apache.spark.sql.auron.NativeConverters$.convertDataType(NativeConverters.scala:205)
        at org.apache.spark.sql.auron.NativeConverters$.convertField(NativeConverters.scala:215)
        at org.apache.spark.sql.auron.NativeConverters$.$anonfun$convertSchema$1(NativeConverters.scala:221)
        at scala.collection.Iterator.foreach(Iterator.scala:943)
        at scala.collection.Iterator.foreach$(Iterator.scala:943)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
        at scala.collection.IterableLike.foreach(IterableLike.scala:74)
        at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
        at org.apache.spark.sql.types.StructType.foreach(StructType.scala:102)
        at org.apache.spark.sql.auron.NativeConverters$.convertSchema(NativeConverters.scala:221)
        at org.apache.spark.sql.execution.auron.plan.Util$.getNativeSchema(Util.scala:43)
        at org.apache.spark.sql.auron.AuronConverters$NativeEmptyExec$1.doExecuteNative(AuronConverters.scala:1094)
        at org.apache.spark.sql.auron.NativeSupports.doExecute(NativeSupports.scala:26)
        at org.apache.spark.sql.auron.NativeSupports.doExecute$(NativeSupports.scala:26)
        at org.apache.spark.sql.auron.AuronConverters$NativeEmptyExec$1.doExecute(AuronConverters.scala:1086)
        at org.apache.spark.sql.execution.SparkPlan.$anonfun$execute$1(SparkPlan.scala:194)
        at org.apache.spark.sql.execution.SparkPlan.$anonfun$executeQuery$1(SparkPlan.scala:232)
        at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
        at org.apache.spark.sql.execution.SparkPlan.executeQuery(SparkPlan.scala:229)
        at org.apache.spark.sql.execution.SparkPlan.execute(SparkPlan.scala:190)
        at org.apache.spark.sql.execution.exchange.ShuffleExchangeExec.inputRDD$lzycompute(ShuffleExchangeExec.scala:135)
        at org.apache.spark.sql.execution.exchange.ShuffleExchangeExec.inputRDD(ShuffleExchangeExec.scala:135)
        at org.apache.spark.sql.execution.exchange.ShuffleExchangeExec.mapOutputStatisticsFuture$lzycompute(ShuffleExchangeExec.scala:140)
        at org.apache.spark.sql.execution.exchange.ShuffleExchangeExec.mapOutputStatisticsFuture(ShuffleExchangeExec.scala:139)
        at org.apache.spark.sql.execution.exchange.ShuffleExchangeLike.$anonfun$submitShuffleJob$1(ShuffleExchangeExec.scala:68)
```

<img width="628" height="628" alt="Image" src="https://github.com/user-attachments/assets/bde9139f-b79b-492f-bd08-370ea790b7e6" />

--- a/spark-extension/src/main/scala/org/apache/spark/sql/auron/AuronConverters.scala
+++ b/spark-extension/src/main/scala/org/apache/spark/sql/auron/AuronConverters.scala
@@ -1090,8 +1090,12 @@ object AuronConverters extends Logging {
         extends LeafExecNode
         with NativeSupports {
 
+      private def nativeSchema = Util.getNativeSchema(output)
+
+      // check whether native converting is supported
+      nativeSchema
+
       override protected def doExecuteNative(): NativeRDD = {
-        val nativeSchema = Util.getNativeSchema(output)
         val partitions = Range(0, outputPartitioning.numPartitions)
           .map(i =>
             new Partition {

